src/lib/stitching/causality/stitching.ts

```diff
@@ -27,16 +27,50 @@ export const causalityStitchingEnvironment = ({
 }: {
   causalitySchema: GraphQLSchema & { transforms: any }
   localSchema: GraphQLSchema
+  version: number
```

Comment:
Given that this is all moved into the stitching file we can probably drop the version check

```suggestion
```